corehq/apps/app_manager/models.py

```diff
@@ -2609,6 +2613,15 @@ def get_case_errors(self, needs_case_type, needs_case_detail, needs_referral_det
             for error in errors:
                 yield error
 
+    def validate_for_build(self):
```

Comment:
If you enable the feature flag you'll get a drop down menu when adding a module and one of the options will be care plan.